hr_expense_invoice/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Supplier invoices on HR expenses',
+    'version': '11.0.1.0.0',
+    'category': 'Human Resources',
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'website': 'https://www.tecnativa.com',
```

Comment:
@cubells Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)